src/private/ptzExtension.ts

```diff
@@ -0,0 +1,296 @@
+    ControlNoResponse,
+    ControlBusy,
+    AckTimeout,
+    ControlTerminated,
+    ControllerTerminated,
+    DataChannelError,
+    ControllerCancelled,
+    ControlDisabled,
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Fetch the ptz capable participants in a meeting.
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * result can either contain the PtzCapableParticipants object, incase of a successful fetch or null when it fails
+   * result: object that contains an array of ptz-capable participants
+   */
+  export function getCapableParticipants(callback: (error: SdkError, result: PtzCapableParticipants) => void): void {
+    if (!callback) {
+      throw new Error('[ptzExtension.getPtzCapableParticipants] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('ptzExtension.getPtzCapableParticipants');
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Request remote control of a participant's ptz camera
+   * @param participant the near-end participant object, for which the far-end is requesting control.
+   * @param callback Callback contains 2 parameters, error and result.
+   * error can either contain an error of type SdkError, incase of an error, or null when fetch is successful
+   * result can either contain the true/false value, incase of a successful request or null when it fails
+   * result: True means request was accepted and false means request was denied
+   */
+  export function requestRemoteCameraControl(
+    participant: PtzParticipantInput,
+    callback: (error: SdkError, result: boolean) => void,
+  ): void {
+    if (!participant) {
+      throw new Error('[ptzExtension.requestRemotePtzCameraControl] Participant cannot be null or empty');
+    }
+    if (!callback) {
+      throw new Error('[ptzExtension.requestRemotePtzCameraControl] Callback cannot be null');
+    }
+    ensureInitialized();
+    const messageId = sendMessageRequestToParent('ptzExtension.requestRemotePtzCameraControl', [participant]);
+    GlobalVars.callbacks[messageId] = callback;
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   *
+   * Send remote ptz custom command to control the near-end's PTZ camera
+   * @param ptzControlCommand PtzControlCommand specifies the command for controling the PTZ camera
+   * @param callback Callback to invoke when the command response returns.
+   */
+  export function sendRemoteCustomCommand(
+    ptzControlCommand: PtzControlCommand,
+    callback: (error: SdkError) => void,
+  ): void {
+    if (!ptzControlCommand) {
+      throw new Error('[ptzExtension.sendRemotePtzCustomCommand] PtzControlCommand cannot be null');
+    }
+    if (!callback) {
+      throw new Error('[ptzExtension.sendRemotePtzCustomCommand] Callback cannot be null');
```

Comment:
should this be '[ptzExtension.sendRemoteCustomCommand] .....'?

Reply:
Make sure to put the right name, in all other places aswell, after you confirm that, it needs tobe changed!!